zebrad/tests/common/get_block_template_rpcs/get_block_template.rs

```diff
@@ -27,9 +30,18 @@ use crate::common::{
 /// We've seen it take anywhere from 1-45 seconds for the mempool to have some transactions in it.
 pub const EXPECTED_MEMPOOL_TRANSACTION_TIME: Duration = Duration::from_secs(45);
 
+/// Delay between getting block proposal results and cancelling long poll requests.
+pub const EXTRA_LONGPOLL_WAIT_TIME: Duration = Duration::from_millis(150);
```

Comment:
This is shorter than the delays we've seen on busy CI runners:
https://github.com/ZcashFoundation/zebra/blob/dc43dca06f8eed275235110e57e643caef0daa26/zebra-test/src/mock_service.rs#L82

If our tests will fail if the test process is paused for more than this time, then we should set it to `DEFAULT_MAX_REQUEST_DELAY`. (We do this in other tests so they automatically adapt when we fix failures due to slow CI machines.)

If it needs to be lower than `BLOCK_PROPOSAL_INTERVAL`, then we might want to increase both.

Either way, can you document why this value is set to this particular amount, and what relationship it needs with `BLOCK_PROPOSAL_INTERVAL`? If it needs to be less, you could add an assert in the test that uses it to make sure.

Reply:
> This is shorter than the delays we've seen on busy CI runners:

I'm not sure if the delay is necessary at all, validating a block proposal may take longer than creating a template, so this should perhaps be a minimum time rather than an added delay, but in any case this test uses `deploy-gcp-tests` in CI. I thought it was a Github runner and went with 10x the expected response time for a block template request originally (deserialization will take some time too), but I think 3x is okay for GCP?

> If it needs to be lower than BLOCK_PROPOSAL_INTERVAL, then we might want to increase both.

It does, I think we want to increase `BLOCK_PROPOSAL_INTERVAL` so that `try_validate_block_template` runs at consistent intervals and is useful, but that this is okay if it's running on a fast GCP compute instance.

> Either way, can you document why this value is set to this particular amount, and what relationship it needs with BLOCK_PROPOSAL_INTERVAL?

Yeah, these constants need definitely better explanations.

Reply:
> I thought it was a Github runner and went with 10x the expected response time for a block template request originally (deserialization will take some time too), but I think 3x is okay for GCP?

At the moment these test runners have lots of CPUs, so we should be getting reasonably good latency. I think they are dedicated machines, but I am not sure, Gustavo would know.